src/Components/Generic/EncounterSheet/index.js

```diff
@@ -33,12 +39,77 @@ const EncounterSheet = ({
     {
       text: `${patient.firstName} ${patient.lastName} ${
         !isTabletMode ? `${t('Encounter date')}: ` : ''
-      } ${moment(encounter.startTime).format(formatDate)}`,
+      } ${Moment(encounter.startTime).format(formatDate)}`,
       separator: false,
       url: '#',
     },
   ];
 
+  useEffect(() => {
```

Comment:
I have a some thoughts about the location of this code, let's talk tomorrow...

Reply:
We talked you move this code into internal component in the PatientBackground and in the future task you connect it to redux, do not?

Reply:
Fixed